src/lib.rs

```diff
@@ -86,8 +92,8 @@ impl PhaseVocoder {
             sum_phase: vec![vec![0.0; frame_size]; channels],
             output_accum: vec![VecDeque::new(); channels],
 
-            forward_fft: rustfft::FFT::new(frame_size, false),
-            backward_fft: rustfft::FFT::new(frame_size, true),
+            forward_fft: planner_forward.plan_fft(frame_size),
+            backward_fft: planner_backward.plan_fft(frame_size),
 
             window: apodize::hanning_iter(frame_size).collect(),
```

Comment:
I think the reason you were unable to get perfect reconstruction is because I didn't really know what I was doing when I wrote this and used a Hanning window both on the input & output. This results in the output being distorted by the square of the window. These days I use the square root of the Hanning window to avoid this. Try:
`window: apodize::hanning_iter(frame_size).map(|x| x.sqrt()).collect(),`

Reply:
In any case just by looking at the code it doesn't seem anything could possibly have been broken by this change. When I have some time I can test it out with https://github.com/nwoeanhinnogaehr/pvoc-plugins to be certain everything still sounds ok

Reply:
Thanks for your comments.

I've followed your suggestion and did a manual test. Reconstructing a "hat wave" or "triangle wave" looks as follows (see screenshot below). The curved thing at the beginning is to be expected (because there's only one window there and you need more overlapping windows). The gap at end is probably an edge effect because the reconstruction stops too early, which, in my opinion, is probably unrelated to my changes. I tried with a longer triangle (not shown) and that also has such a gap; not at the same place, but at the end, which confirms my idea.

![pvoc-test](https://user-images.githubusercontent.com/21333083/75090994-e7138f80-5568-11ea-83b6-e369e3f19bdd.png)